code/datums/mutations/body.dm

```diff
@@ -128,10 +132,68 @@
 	REMOVE_TRAIT(owner, TRAIT_DWARF, GENETIC_MUTATION)
 	owner.visible_message(span_danger("[owner] suddenly grows!"), span_notice("Everything around you seems to shrink.."))
 
+/datum/mutation/human/acromegaly
+	name = "Acromegaly"
+	desc = "A mutation believed to be the cause of acromegaly, or 'being unusually tall'."
+	quality = MINOR_NEGATIVE
+	difficulty = 16
+	instability = -10
+	synchronizer_coeff = 1
+	conflicts = list(/datum/mutation/human/dwarfism)
+
+/datum/mutation/human/acromegaly/on_acquiring(mob/living/carbon/human/owner)
+	if(..())
+		return
+	ADD_TRAIT(owner, "TRAIT_TOO_TALL", GENETIC_MUTATION)
```

Comment:
You've defined `TRAIT_TOO_TALL` but you wrap it in a string in every place you use it
```suggestion
	ADD_TRAIT(owner, TRAIT_TOO_TALL, GENETIC_MUTATION)
```